docs/config.md

```diff
@@ -43,9 +43,11 @@ We support all Vault Environment Variables listed [here](https://www.vaultprojec
 | Name            | Description | Notes |
 | --------------- | ----------- | ----- |
 | AVP_TYPE           | The type of Vault backend  | Supported values: `vault`, `ibmsecretsmanager`, `awssecretsmanager` and `gcpsecretmanager` |
+| VAULT_ADDR         | The Vault URL              | The Vault address (e.g. `http://0.0.0.0:8200`)                                             |
```

Comment:
We shouldn't need to add this here as above we link directly to the Vault Env Var list